mathematical_puzzles/Q1/Main.scala

```diff
@@ -0,0 +1,20 @@
+import scala.util.control.Breaks
+
+object Main extends App{
+  var i:Int = 10
+
+  while(true){
+    if(isPalindromes(Integer.toString(i)) && isPalindromes(Integer.toOctalString(i)) && isPalindromes(Integer.toBinaryString(i))){
+      println(i)
+      val b = new Breaks
+      b.break()
+    }else{
+      i+=1
+    }
+  }
```

Comment:
And I don't recommend `scala.util.control.Breaks` since it has big performance penalties because of its implementation.  If you can, using higher order functions are better solution:

```scala
// Iterator.from(10, 1) creates infinite iterator, start is 10, step is 10
val answer = Iterator.from(10, 1).filter{i =>
 // Iterator#filter returns new Iterator holds only elements that satisfy a predicate
 // Since filter is lazily called, you don't need worry about too much memory consumption
  isPalindromes(Integer.toString(i)) && 
  isPalindromes(Integer.toOctalString(i)) && 
  isPalindromes(Integer.toBinaryString(i))
}.take(1) // take the first element that satisfy the predicate
println(answer)
```